tests/test_iib_ops.py

```diff
@@ -293,7 +294,7 @@ def remove_operators_mock_calls_tester(
     "extra_args,push_items,mock_calls_tester",
     [
         (
-            [],
+            ["--deprecation-list", ["bundle1"]],
```

Comment:
Shouldn't this be `"--deprecation-list", "bundle1"`

Reply:
Could you also add a test where there are more than one bundles in the deprecation_list?

Reply:
added